pyedflib/edfwriter.py

```diff
@@ -978,7 +984,7 @@ def _calculate_optimal_record_duration(self):
         assert record_duration<=60, 'record duration must be below 60 seconds'
         self.record_duration = record_duration
 
-    def _get_sample_frequency(self, channelIndex):
+    def _get_sample_frequency(self, channelIndex: int) -> Optional[Union[int, float]]:
```

Comment:
```suggestion
    def _get_sample_frequency(self, channelIndex: int) -> Union[int, float]:
```

should always return something